packages/govuk-frontend-review/src/views/layouts/layout.njk

```diff
@@ -14,7 +14,14 @@
     <script src="/vendor/iframe-resizer/iframeResizer.min.js" defer></script>
     <script>
       document.addEventListener('DOMContentLoaded', function () {
-        window.iFrameResize({}, '.js-component-preview')
+        var $examples = document.querySelectorAll('.js-component-preview')
+
+        // Resize component preview iframes when loaded
+        for (var i = 0; i < $examples.length; i++) {
+          $examples[i].addEventListener('load', function () {
```

Comment:
There's a lot more layout shift on the review app than the DS website - we've set some height classes there so that the unloaded examples take up roughly the right amount of space. Probably not a biggie for an internal app, and it looks like it'd be trickier to target above-fold iframes and make 'em eager here than on the DS site, so not a blocker for this PR.

Reply:
Yeah it's a bit jarring isn't it

Can we keep the first 3x components with `loading="eager"` to keep the old behaviour? Pushed 👍

We have a lot of internal margin/padding inside our iframes, but works for smaller components like "Button"

Originally [previews just had `margin: 15px 0;`](https://github.com/alphagov/govuk-frontend/blob/80e1b90caed2f91f1c2f34540ee606409924483b/packages/govuk-frontend-review/src/stylesheets/partials/_app.scss#L7) but then we added `.govuk-main-wrapper--auto-spacing` and the "**Code**" `<details>` got pushed much further down and the shift is quite prominent